utils/mil_tools/nodes/online_bagger.py

```diff
@@ -148,8 +160,11 @@ def subscribe_loop(self):
             i = i + 1
             if i % 1000 == 0:
                 rospy.logdebug('still subscribing!')
+        rospy.loginfo("Subscribed to {} of {} topics, will try again every {} seconds".format(self.successful_subscription_count, 
```

Comment:
Just for your information, you didn't need str.format() here because loginfo is already a formatter. You could've equivalently written:
`rospy.loginfo("Subscribed to %d of %d topics, will try again every %d seconds", self.successful_subscription_count, len(self.subscriber_list), self.resubscribe_period)`
This is true for all the rospy logger functions.